json-form-react/JsonForm.js

```diff
@@ -53,22 +53,46 @@ const JsonForm = (props) => {
     const fieldClasses = `jsfr-field ${field.class} ${formErrors[field.name] ? 'jsfr-field-error' : ''}`;
     const fieldWrapperClasses = `jsfr-field-wrapper ${field.wrapperClass} ${formErrors[field.name] ? 'jsfr-field-wrapper-error' : ''}`;
 
-    return (
-      <div key={field.name} className={fieldWrapperClasses}>
-        <label htmlFor={fieldId}>
-          {field.label}
-        </label>
-
-        <input
-          id={fieldId}
-          type={field.type}
-          name={field.name}
-          value={formData[field.name] || ''}
-          placeholder={field.placeholder}
-          className={fieldClasses}
-          onChange={onChange}/>
-      </div>
-    );
+    if (field.type == 'radio') {
+      return (
+        <div key={field.name} className={fieldWrapperClasses}>
+          <label htmlFor={fieldId}>
+            {field.label}
+          </label>
+          <label >
+            <br/>
+                male    female
+            <br/>
+
+          </label>
+          {Object.keys(field.option).map((keys) => {
+
+            return (
+              <input type="radio" key={keys} id={field.option[keys]} name="gender" value={field.option[keys]}/>
```

Comment:
@yasikahivin To avoid the key error, let's get an index from the `map` function and use it.
You can get the index like:
`{Object.keys(field.options).map(keys, idx) => { ... }`

Reply:
noted @cie6868